### PR TITLE
Stuff

### PR DESCRIPTION
**Pull request by AnyBananaGAME**

### CODE REVIEW

**Comment by Liqee431** [comment]
New kickcommand but there is ni kickcommand?

**Comment by andriycraft** [comment]
There is…

**Comment by AnyBananaGAME** [comment]
> New kickcommand but there is no kickcommand?

No, did not add any new KickCommand stuff... I just added "," at the end of new KickCommand()
Before: new KickCommand() 
After: new KickCommand(), 

If i did not add that it would cause an error and just crash.

**Comment by AnyBananaGAME** [comment]
And there is a KickCommand, It's located in the default commands folder

File: start.bat [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@echo off
+echo The server is being started!! NOTE: The server will automatically restart on crash/kill!
+:main
+node start.js
+echo The server either CRASHED or was killed... Starting a force restart.
```
Comment (andriycraft) [inline start.bat:5]: Or stopped

**Review by andriycraft — CHANGES_REQUESTED** [review]

File: start.bat (hunk 1 above)
Comment (PiggyPlex) [reply start.bat:5]: should have a `goto main` statement to force restart

Comment (PiggyPlex) [reply start.bat:5]: should have a `timeout /t 2 /nobreak > nul` to be consistent with `start.sh` script

File: src/command/default/FakeMessageCommand.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                sender.sendMessage(TextFormat.RED + "/fakesay <Username> <message>");
+            }
+            return;
+        }
+        sender.getServer().broadcastMessage(TextFormat.WHITE + `<${args[0]}> ` + args.slice(1).join(' '));
+        console.log(`<${args[0]}> ${args.slice(1).join(' ')}`)
+    }
+}
+
+module.exports = FakeSayCommand;
```
Comment (PiggyPlex) [inline src/command/default/FakeMessageCommand.js:22]: is this really needed as a core command?

**Review by PiggyPlex — CHANGES_REQUESTED** [review]

File: src/command/default/FakeMessageCommand.js (hunk 2 above)
Comment (AnyBananaGAME) [reply src/command/default/FakeMessageCommand.js:22]: No, but can be used

**Comment by VaxPex** [comment]
> > New kickcommand but there is no kickcommand?
> 
> No, I did not add any new KickCommand stuff... I just added "," at the end of `new KickCommand()` Before: `new KickCommand() ` After: `new KickCommand(), `
> 
> If i did not add that it would cause an error and just crash.

It wont

**Comment by andriycraft** [comment]
vaxpex is alive

**Comment by andriycraft** [comment]
> > > New kickcommand but there is no kickcommand?
> > 
> > 
> > No, I did not add any new KickCommand stuff... I just added "," at the end of `new KickCommand()` Before: `new KickCommand() ` After: `new KickCommand(), `
> > If i did not add that it would cause an error and just crash.
> 
> It wont

It will kick the Player that used this command

**Comment by AnyBananaGAME** [comment]
Ye, gives server an error and crashes the player.

**Comment by PiggyPlex** [comment]
https://github.com/BlueBirdMC/BlueBird/pull/29/files/bfe5025a772e425a7cd87f684aad76f6b6799b9d

**Pull request closed without merging**